packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/__tests__/cc44.test.tsx

```diff
@@ -153,6 +153,92 @@ describe("Indemnité licenciement - CC 44", () => {
     });
   });
 
+  describe("parcours avec la convention collective pour valider ses spécificités SANS PREAVIS", () => {
+    beforeEach(() => {
+      render(
+        <CalculateurIndemnite
+          icon={""}
+          title={""}
+          displayTitle={""}
+          slug={"indemnite-licenciement"}
+        />
+      );
+      userEvent.click(ui.introduction.startButton.get());
+      userEvent.click(ui.contract.type.cdi.get());
+      userEvent.click(ui.contract.fauteGrave.non.get());
+      userEvent.click(ui.contract.inaptitude.non.get());
+      userEvent.click(ui.contract.arretTravail.non.get());
+      userEvent.click(ui.next.get());
+      userEvent.click(ui.next.get());
+      userEvent.selectOptions(
+        ui.information.agreement44.proCategory.get(),
+        "Ouvriers et collaborateurs (Groupes I à III)"
+      );
+      fireEvent.change(ui.information.agreement44.age.get(), {
+        target: { value: "40" },
+      });
+      userEvent.click(ui.next.get());
+      fireEvent.change(ui.seniority.startDate.get(), {
+        target: { value: "01/01/2000" },
+      });
+      fireEvent.change(ui.seniority.notificationDate.get(), {
+        target: { value: "01/01/2023" },
+      });
+      fireEvent.change(ui.seniority.endDate.get(), {
+        target: { value: "01/01/2023" },
+      });
+      userEvent.click(ui.seniority.hasAbsence.non.get());
+      userEvent.click(ui.next.get());
+      // Validation que l'on est bien sur l'étape ancienneté
+      expect(ui.activeStep.query()).toHaveTextContent("Salaires");
+    });
+
+    test("Vérification que la cc est privilégié grâce à un salaire de référence conventionnel import", () => {
```

Comment:
```suggestion
    test("Vérification que la cc est privilégiée grâce à un salaire de référence conventionnel", () => {
```

Je dirai que le mot import s'est glissé par erreur non ? Par contre il manque peut être quelque chose dans le titre car je n'ai pas compris 🤔